unix/datetimepicker.c

```diff
@@ -111,8 +122,11 @@ static void setLabel(dateTimePickerWidget *d)
 
 static void dateTimeChanged(dateTimePickerWidget *d)
 {
+	uiDateTimePicker *c;
```

Comment:
Honestly, I would make this emit an actual signal and separate the widget implementation from the uiDateTimePicker implementation. But if you can't figure out how to do that, don't worry (I'll probably take care of that instead).

Reply:
Will emit a GTK+ signal like a proper widget then. Agreed that is better separation of concerns.

Reply:
Will emit a GTK+ signal like a proper widget then. Agreed that is better separation of concerns.